contrib/utilities/relaunch_simulations.sh

```diff
@@ -0,0 +1,312 @@
+#     file in the folder.
+#   - A job script for launching the simulation on the cluster.
+# - All parameter files must contain already the "restart" subsection if they
+#   have to be restarted from a checkpoint.
+# - All job scripts should have the same name even if their content is different
+#   (e.g. launch.sh or job.sh)
+# - All job output log files must have the suffix ".out" and have the job ID
+#   as part of their name.
+#
+# Tree:
+#   .
+#   ├── relaunch_simulations.sh
+#   ├── <search_file>
+#   ├── <simulation_00_folder>
+#   │   ├── <job_file>
+#   │   └── <parameter_file>
+#   ├── <simulation_01_folder>
+#   │   ├── <job_file>
+#   │   └── <parameter_file>
+#   ├── <simulation_02_folder>
+#   │   ├── <job_file>
+#   │   └── <parameter_file>
+#   ...
+#
+# How does the script work:
+# - A search file (-sf|--search_file) and job file name (-j|--job_file) must be
+#   specified when calling the script. The search file contains the name of the
+#   simulation folders to be checked (1 folder = 1 line in the search file).
+#   Example content of a search file:
+#     simulation_00_folder
+#     simulation_01_folder
+#     simulation_02_folder
+#     ...
+# - The script checks the status of the latest launched job of the simulations
+#   in the search file.
+#   - If the status is "CANCELLED", check the previous job until a
+#     non-"CANCELLED" simulation is found.
+#     This way, user-stopped simulations are ignored.
+# - If the job was found to be "TIMEOUT":
+#   - The simulation folder name is saved in a file named "timeout_simulations.txt".
+#   - If the user requested that the simulation should be rerun from checkpoint
+#     with (-lr|--launch_restart), then the scripts searches for the "set restart"
+#     in the "subsection restart" of the parameter file and changes its value to
+#     "true". Then, it launches the simulation with the specified job file.
+#     The launched simulation folder name is saved into a file named
+#     "relaunched_simulations.txt". The idea is to reuse this file as the
+#     "search_file" when relaunching again. At the second iteration, to avoid
+#     overwriting "relaunched_simulations.txt" and losing track of the previous
+#     relaunch, a copy can be made by adding the argument (-cp|--copy_search_file)
+#     to the script call.
+# - If the job was found to be "FAILED":
+#   - The simulation folder name is saved in a file named "failed_simulations.txt".
+#   - If the user requested that the failed simulation should be rerun with
+#     (-rf|--rerun_failed), then the simulation is rerun without restart.
+#   - If the user wishes to rerun the simulation from the last checkpoint, the
+#     (-st|--set_restart_true) flag must be added to the call.
+#   - All relaunched simulations, from a checkpoint or not, are listed in the
+#     "relaunched_simulations.txt" file.
+#
+# For help on the different arguments and their default values, the user can
+# call the script with the flag (-h|--help).
```

Comment:
Do you think you could provide a quick example with the above folder structure. Like just a use case example, that would really help users understand.
However, that's already a very cool and clear explanation. Good documentation it's dope.